### ISSUE TITLE
Settle `iroh start` arguments

### ISSUE THREAD

**Issue by Frando** [n0-computer/iroh#1403]
After #1356 is merged, we should have have a look at how we want `iroh start` (which was `iroh provide`) before to behave. Currently the next argumetn is either treated as a path, or `STDIN` is added as a blob.

Likely we want to leave adding `STDIN` to the separate `add` command. And to add a folder on start, maybe better have a `--add` argument (could have be multiple).

**Comment by dignifiedquire** [comment]
Some if this was already done, for more discussions on this https://github.com/n0-computer/iroh/discussions/1659
